syntax = "proto3";
package xbospb;
import "nullabletypes.proto";
import "weather_current.proto";
message Weather_Prediction_State {
    // time of the prediction (leave empty
    // unit: ns

    message Prediction {
        uint64 prediction_time = 1;
        Weather_Current_State prediction = 2;
    }
    repeated Prediction predictions = 1;
}
